modules/virt-enlarging-vm-disk.adoc

```diff
@@ -0,0 +1,88 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virtual_disks/virt-vm-disk-resizing.adoc
+
+:_content-type: PROCEDURE
+[id="virt-enlarging-vm-disk_{context}"]
+= Enlarging a virtual machine disk
+
+You can enlarge a virtual machine (VM) disk by resizing the disk's persistent volume claim (PVC).
+
+VM disk enlargement makes extra space available to the virtual machine. However, it is the responsibility of the VM owner to decide how to consume the storage.
+
+Enabling VM disk enlargement notifies the virtual machine about size changes and, if the disk is a `Filesystem` PVC, the matching file expands to the remaining size while reserving some space for file system overhead.
```

Comment:
Use active voice.

I'd suggest getting rid of the first part of the sentence and rewriting this as "If the disk is a `Filesystem` PVC, the matching file expands to the remaining size while reserving some space for file system overhead."